index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1 class="header">freeCodeCamp Survey Form</h1>
+    <p class="para-description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+    <form>
+      <div class="form-container">
+        <label id="name" for="name">Name</label>
+        <input
+          type="text"
+          name="name"
+          id="input-name"
+          class="form-input"
+          placeholder="Enter your name"
+        />
+      </div>
+      <div class="form-container">
+        <label id="name" for="name">Email</label>
+        <input
+          type="text"
+          name="name"
+          id="input-name"
+          class="form-input"
+          placeholder="Enter your email"
+        />
+      </div>
+      <div class="form-container">
+        <label id="name" for="name">Age (Optional)</label>
+        <input
+          type="text"
+          name="name"
+          id="input-name"
+          class="form-input"
+          placeholder="Age"
+        />
+      </div>
+      <div class="form-container">
+        <label id="name" for="name"
+          >Which option best describes your current role?</label
+        >
+        <select name="current-role" id="current-rolw" class="form-input">
+          <option value="student">Student</option>
+          <option value="fulltimejob">Full Time Job</option>
+          <option value="fulltimelearner">Full Time Learner</option>
+          <option value="prefernottosay">Prefer not to say</option>
+          <option value="other">Other</option>
+        </select>
+      </div>
```

Comment:
Kindly, implement the correct spacing in your form. Good padding and margins make your website look nice and appealing to the users. 
 
![test1](https://user-images.githubusercontent.com/64077816/165069187-86da803b-48ca-4669-9bed-eef10116e0ab.png)

Reply:
- [ ] Nicely done 👍